### PR TITLE
new conda forge package cp2kdata

### PR DESCRIPTION
**Pull request by robinzyb**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by robinzyb** [comment]
@conda-forge/staged-recipes @conda-forge/help-python Ready for review!

File: recipes/cp2kdata/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - cp2kdata
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/robinzyb/cp2kdata
+  summary: A Small Package to Postprocess Cp2k Output
+  license: LGPL-3.0
```
Comment (njzjz) [inline recipes/cp2kdata/meta.yaml:50]: Should be

```suggestion
  license: LGPL-3.0-only
```

or

```suggestion
  license: LGPL-3.0-or-later
```

See https://spdx.org/licenses/

File: recipes/cp2kdata/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    - click
+    - regex
+    - monty
+    - pyyaml
+    - asciichartpy
+
+test:
+  imports:
+    - cp2kdata
+  commands:
```
Comment (synapticarbors) [inline recipes/cp2kdata/meta.yaml:42]: Please test the entry point cli:
https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#test-commands

Comment (robinzyb) [reply recipes/cp2kdata/meta.yaml:43]: I added the command test

Comment (synapticarbors) [reply recipes/cp2kdata/meta.yaml:43]: It looks like you deleted the `pip check` command, which you should put back in.

Comment (robinzyb) [reply recipes/cp2kdata/meta.yaml:43]: sorry, thought it was a placeholder or something else.

**Pull request merged**